capstone/src/main/java/com/google/univiz/SuggestionData.java

```diff
@@ -1,4 +1,18 @@
 package com.google.univiz;
 
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import javax.annotation.Nullable;
+
 /** SuggestionData represents the college data specific to the Search feature */
-public class SuggestionData {}
+@GenerateTypeAdapter
+@AutoValue
+abstract class SuggestionData {
+  @SerializedName("school.name")
+  @Nullable
+  abstract String name();
+
+  @SerializedName("id")
+  abstract int id();
```

Comment:
I guess this works for now, I doubt any code we have that deserializes into a CollegeId, and such a feature doesn't seem necessary.